src/tests-rosetta/api.ts

```diff
@@ -2206,7 +2206,7 @@ describe('Rosetta API', () => {
   // TODO: fails with:
   //  Response 500: Internal Server Error fetching http://127.0.0.1:20443/v2/pox - Failed to query peer info
   //  https://github.com/blockstack/stacks-blockchain/issues/2600
-  test.skip('construction/metadata - delegate-stacking', async () => {
+  test.skip('construction/metadata - delegate_stacking', async () => {
```

Comment:
@asimm241 looks like this might have been fixed by https://github.com/blockstack/stacks-blockchain/pull/2634, can we try not skipping it for additional test coverage?